web_hidden_field/models/hidden_template.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Ignacio Ibeas <ignacio@acysos.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import models, fields
+
+
+class HiddenTemplateField(models.Model):
+    _name = 'hidden.template.field'
+    _description = 'Hidden template field'
+
+    name = fields.Many2one(
```

Comment:
would be better to rename this field to `field_id` and use `_rec_name = 'field_id'`